modules/search-service-api/src/main/java/org/opencastproject/search/api/SearchResultItemImpl.java

```diff
@@ -63,6 +64,9 @@
   @XmlElement(name = "mediapackage", namespace = "http://mediapackage.opencastproject.org")
   private MediaPackage mediaPackage = null;
 
+  // TODO: should this be serialized?!
```

Comment:
We don't need this, this would cause more traffic since the REST interfaces would generate more data and it's easy to add later on → I wouldn't serialize this for now.

Reply:
Thinking about it again, I think we need it so that the search service remote impl works. That seems to work by using the standard search API. Is that correct?

Reply:
So I am fairly sure this is required in order for the remote impl to work. Take a look here: https://github.com/opencast/opencast/blob/418fe15ded66b4491c1f5dc72b209f96d4c16ae8/modules/search-service-remote/src/main/java/org/opencastproject/search/remote/SearchServiceRemoteImpl.java#L154

So the remote impl sends a request to the normal search API and deserializes the XML. If we don't serialize this field, it would be missing.